src/style.css

```diff
@@ -1,3 +1,103 @@
+/* ======= VARS ======= */
+
+:root {
+  /* colors */
+  --black: #000;
+  --grey: #666;
+  --yellow: #f0f4e7;
+  --violet: #e3d9ed;
+  --white: #fff;
+  --blue: #00acdc;
+
+  /* font-size */
+  --main: 12px;
+  --line-height: 14px;
+}
+
+/* ======= NULL CSS ======= */
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: var(--main);
+  color: var(--black);
+  background-color: var(--white);
+}
+
 body {
   margin: 0;
 }
+
+ul,
+li {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a,
+a:visited {
+  text-decoration: none;
+}
+
+/* =======HEADER ======= */
+
+.header__wraper {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+.header__list {
+  display: flex;
+}
+
+.header__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.header__link {
+  font-weight: 500;
+  font-size: var(--main);
+  line-height: 60px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  color: var(--black);
+  height: 60px;
+  transition: color 0.3s ease;
+}
+
+.header__link:hover {
+  color: var(--blue);
+}
+
+.header__link:active {
+  color: var(--blue);
```

Comment:
You can combine the selectors with same properties

Reply:
done